Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -1611,6 +1611,11 @@ bool WebAnimation::computeRelevance()
     if (m_playbackRate < 0 && timing.phase == AnimationEffectPhase::After)
         return true;
 
+    // - the animation effect is associated with an animation not in the idle play state with a non-null
```

Comment:
Can you add a link to https://drafts.csswg.org/web-animations-1/#current here (or somewhere in this function)?